info.xml

```diff
@@ -15,11 +15,12 @@
     <url desc="Licensing">http://www.gnu.org/licenses/agpl-3.0.html</url>
   </urls>
   <releaseDate>2018-03-06</releaseDate>
-  <version>2.0</version>
+  <version>2.1</version>
   <develStage>stable</develStage>
   <compatibility>
     <ver>4.7</ver>
     <ver>5.0</ver>
+    <ver>5.8</ver>
```

Comment:
Why do we need this? Can we indicate something like 5.0+ ? Also, I think that Eileen has removed 4.7 from new releases of her extensions. People will still be able to get versions that are compatible with 4.7, but she doesn't want to be forced to support them. We might want to do the same...do you want to think about this?

Reply:
My bad we now change the existing version to the latest i.e. 5.0 should be changed to 5.11 not 5.0+. Heres an example https://github.com/eileenmcnaughton/nz.co.fuzion.extendedreport/blob/master/info.xml#L20

Reply:
Would the extension work on every release in 4.7.x and 5.x that we know of? If so, in order to encourage adoption, let's on this extension mark current release as supporting all versions until we have something that doesn't work except with newer releases.

Reply:
@JoeMurray yes, it does. I have just quickly confirmed on my local so it would be ok to mark extension to the current release.